cmd/libs/go2idl/make-client/output/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I think we have a rule requiring a doc.go for each package, though not sure why we have the rule.